src/quicktype-core/input/JSONSchemaStore.ts

```diff
@@ -3,6 +3,8 @@ import { StringMap, assert } from "../support/Support";
 export type JSONSchema = StringMap | boolean;
 
 export abstract class JSONSchemaStore {
+    constructor(protected headers?: string[]) {}
```

Comment:
I could see moving this to the `FetchingJSONSchemaStore` constructor. Might be better there.

Reply:
Done.